includes/class-wp-vote-db.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * WP_Vote_DB class
+ *
+ * @package ubc_wp_vote
+ * @since 0.0.1
+ */
+
+namespace UBC\CTLT\WPVote;
+
+/**
+ * Database wrapper class for UBC WP Vote plugin.
+ * This class talks directly to WordPress database and provide API for other logic layer.
+ *
+ * @since 0.0.1
+ */
+class WP_Vote_DB {
+
+	/**
+	 * Create global database tables that required by this plugin
+	 *
+	 * @since 0.0.1
+	 */
+	public static function create_tables() {
+		global $wpdb;
+		$charset_collate = $wpdb->get_charset_collate();
+		$table_name = $wpdb->base_prefix . 'ubc_wp_vote';
```

Comment:
Let's sanitize this. We've probably got bigger problems if $wpdb->base_prefix is compromised, but let's not let our little plugin be another method for bad actors to use.

Reply:
Also sanitize the $charset_collate variable appropriately too, please.